### PR TITLE
fix: change default values for Ability() (cosmetic + avoid NoneType error)

### PR DESCRIPTION
**Pull request by zaphodef**
Using `variations=None` by default was leading to an error when iterating over it.

### CODE REVIEW

**Comment by zaphodef** [comment]
Also fixed the same issue for `parsers` and `requirements`

File: app/objects/c_ability.py [hunk 1]
```diff
@@ -55,12 +55,15 @@ def __init__(self, ability_id, tactic=None, technique_id=None, technique=None, n
...
-        self.parsers = parsers
-        self.requirements = requirements
+        self.parsers = parsers if parsers else []
+        self.requirements = requirements if requirements else []
         self.privilege = privilege
         self.timeout = timeout
         self.repeatable = repeatable
-        self.variations = [Variation(description=v['description'], command=v['command']) for v in variations]
+        if variations:
+            self.variations = [Variation(description=v['description'], command=v['command']) for v in variations]
```
Comment (khyberspache) [inline app/objects/c_ability.py:64]: You can follow the same format as above:
```Python
self.variations = [Variation(description=v['description'], command=v['command']) for v in variations] if variations else []
```

Comment (zaphodef) [reply app/objects/c_ability.py]: Fixed in 083b1cb

**Comment by privateducky** [comment]
@khyberspache  you just get me

**Pull request merged**